.vib/keycloak/cypress/cypress/support/commands.js

```diff
@@ -1,4 +1,4 @@
-const COMMAND_DELAY = 1100;
+const COMMAND_DELAY = 500;
```

Comment:
Not blocker for now but we should unify all the COMMAND_DELAY to ensure all tests are aligned. Waiting 500ms or 1300ms (the maximum value I observed in the repo) is nothing after reducing the number of tests. Tuning too much the delays to reduce them at the maximum, sometimes, can give us false positives (due to extra needed delay). IMHO, it's something non blocker now but should be unified in another PR.

Reply:
Good idea, I created https://github.com/bitnami/charts/pull/11617 and updated this PR accordingly.